e/envoy/Dockerfiles/1.25.3_ubi_8/Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.4
```

Comment:
@sumitd2 - Any particular reason that we are using ubi 8.4?

Reply:
yes, the customer wanted it